src/dotnet/APIView/APIViewWeb/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Octokit;
+
+namespace APIViewWeb.Pages
+{
+    public class IndexModel : PageModel
```

Comment:
I don't think this page, adds a lot of value to the app. You can put the same information into the header.